templates/prometheus_exporter/solrcloud_prometheus_exporter.service.j2

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=solr webserver daemon
```

Comment:
Maybe this could be more descriptive like **solr prometheus exporter daemon**

Reply:
done! (actually, it was a good copy paste :) )